### DISCUSSION TITLE
Add Sandbags/Test Dummies to Test Ride (yellow flag) mode.

### DISCUSSION THREAD

**Discussion by Shima33** [OpenRCT2/OpenRCT2#15777] [Feature requests]
Logically this makes sense, as it's akin to how a real rollercoaster is tested - the simulation is with empty carriages, and then the test runs are with sand bags, to simulate the weight of people. 

In much the same way, RCT2 could, during testing, add a person, at max weight, to every seat of the ride - edited so that they appear as either yellow sandbags or white test dummies; this can be done with in-game palette swaps of the peep sprite without having to design new sprites and their interactions. 

This would be great for rides such as the Ghost Train, Bobsleigh coaster or Side-Friction coaster, as it would allow coaster designers to work out where the problem areas of their ride are before any lives are put at risk, as well as hopefully not being too hard to introduce.

**Comment by James103** [comment]
Maybe a plugin could do what you're looking for in terms of game mechanics? In other words, listen for the ride set status action, and set the mass of each vehicle created as if various people were on the ride (with weight either at max or configurable).

**Comment by duncanspumpkin** [comment]
Its not at all possible to have different images on rides unless you can find every single person who made a ride and request they rerender the whole thing. You can modify a vehicles weight without much effort though.

**Comment by IntelOrca** [comment]
You could provide the functionality through simulation only, using the peep sprites for that would be fine as it is all ghost colour.

**Comment by Shima33** [comment]
> You could provide the functionality through simulation only, using the peep sprites for that would be fine as it is all ghost colour.

This is what I had in mind, yes. Sorry if my original post wasn't worded clearly.
